graph/api/src/main/java/org/jboss/windup/graph/service/GraphService.java

```diff
@@ -181,9 +185,39 @@ protected FramedGraphQuery findAllQuery()
     }
 
     @Override
+    public Iterable<T> findAllByPropertyNotMatchingRegex(final String key, final String... regex)
```

Comment:
this may return 99 percent of all the vertices in case it will be used wrong, right?